frontend/vitest.config.ts

```diff
@@ -28,7 +28,7 @@ export default mergeConfig(
         thresholds: {
           lines: 84,
           // functions: 20, // has problems see https://github.com/vitest-dev/vitest/issues/3607
-          branches: 92,
+          branches: 89,
```

Comment:
Why do we need to change this here?

Reply:
The lines of code have increased mainly by adding the file `tokens.module.scss` but as far as I see there is no sense to test the lines. Therefore the relation of tested lines to total count of lines decreases.

Reply:
This is not true

![image](https://github.com/Ocelot-Social-Community/Ocelot-Social/assets/1238238/2cb38da5-358e-461b-a54f-0e3089bc7cb5)

This part is uncovered and represents new functionality, namely a button, which is untested. This is correctly detected.

I think its valid to reduce coverage tho, just correcting the statement.

![image](https://github.com/Ocelot-Social-Community/Ocelot-Social/assets/1238238/885518c8-7693-432c-9803-21d26fcdd6ee)

Reply:
Sorry for my wrong statement and thanks for correcting me @ulfgebhardt